### PR TITLE
Feature branch for review code

### PR DESCRIPTION
**Pull request by digomes87**

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # frozen_string_literal: true
-RSpec.configure do |config|
 
+RSpec.configure do |config|
   config.expect_with :rspec do |expectations|
     expectations.include_chain_clauses_in_custom_matcher_descriptions = true
```
Comment (samgaco) [inline spec/spec_helper.rb]: :medal_sports:

Comment (digomes87) [reply spec/spec_helper.rb]: I back my version, how you do the test ?

Comment (samgaco) [reply spec/spec_helper.rb]: most things working previously were failing like the methods returning enumerable when no blocks or arguments given, some of the functionalities of the methods when getting classes, etc... but checking your new commits, everything looks fine, so submit the project for another review, you should get the pass :+1:

File: enumerated.rb [hunk 2]
```diff
@@ -1,120 +1,137 @@
...
   end
 end
 
-def multiply_els(arra)
-  arra.my_inject { |accum, item| accum * item }
-end
+# multiply_els 10
+def multiply_els(value)
+  value.my_inject :*
+end
```
Comment (samgaco) [inline enumerated.rb]: :medal_sports:

Comment (digomes87) [reply enumerated.rb]: thank you a lot !!!

**Pull request merged**